### DISCUSSION TITLE
Add dev.morphia.morphia:morphia-core:2.2.7 to mvn etc. mvn

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [MorphiaOrg/morphia#1878] [General]
Can you add to mvn etc.

`implementation("dev.morphia.morphia:morphia-core:2.2.7")`

& 

`earlib group: "dev.morphia.morphia", name: "morphia-core", version: "2.2.7", ext: "jar"
`

https://mvnrepository.com/artifact/dev.morphia.morphia/morphia-core

TIA

**Comment by evanchooly** [comment]
2.2.7 hasn't been released yet.  That's why you can't find it there.

**Comment by NOTiFYcouk** [comment]
I thought "https://github.com/MorphiaOrg/morphia/issues/1876" was referring to "added this to the 2.2.7 milestone 2 days ago" being available. 

Trying to get my EJB working with Morphia @ https://github.com/MorphiaOrg/morphia/discussions/1875.

**Reply by evanchooly** [reply]
That means any fix is targeted for a 2.2.7 release.  But it hasn't been released yet.
